src/volcanic_island_simulator.py

```diff
@@ -59,6 +59,8 @@
 
 _DEFAULT_SEA_LEVEL_PARAMS = {"mean": 0, "amplitude": 100, "period": 10000}
 
+_DFAULT_TECTONIC_PARAMS = {"uplift": 0}  # m/yr of relative uplift (+) or subsidence (-)
```

Comment:
default of zero uplift makes sense to me; rivers will be forced to adjust to sea level changes only